worlds/ladx/__init__.py

```diff
@@ -444,6 +444,10 @@ def stage_assert_generate(cls, multiworld: MultiWorld):
         if not os.path.exists(rom_file):
             raise FileNotFoundError(rom_file)
 
+    def generate_early(self):
+        if self.options.goal == "specific":
+            assert self.options.instrument_count.value > 0, "Specific instrument count cannnot be zero."

```

Comment:
`assert` should not be used for any necessary control flow. It's for debugging and testing.
An optimized build can remove `assert` statements because you shouldn't need debugging or testing in an optimized build.

```suggestion
            if self.options.instrument_count.value <= 0:
                raise ValueError("Specific instrument count cannot be zero.")
```